### PR TITLE
feat: Bump ai-navigator-cluster-info chart version to 0.2.9

### PR DESCRIPTION
**Pull request by rajgitnutanix**
**What problem does this PR solve?**:

Bump ai-navigator-cluster-info chart version to 0.2.9
**Which issue(s) does this PR fix?**:
https://jira.nutanix.com/browse/NCN-104406

### CODE REVIEW

File: licenses.d2iq.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ ignore:
   - docker.io/mesosphere/trivy-bundles:0.56.2-20241104T225447Z
   - docker.io/library/alpine:3.20.3
 
+
```
Comment (alembiewski) [inline licenses.d2iq.yaml:6]:
```suggestion
```

File: licenses.d2iq.yaml [hunk 2]
```diff
@@ -553,7 +554,7 @@ resources:
       - url: https://github.com/kubernetes-sigs/external-dns
         ref: v${image_tag%-debian-12-r7}
         license_path: LICENSE
-  - container_image: docker.io/bitnami/postgresql:15.8.0-debian-12-r14
+  - container_image: docker.io/bitnami/postgresql: 15.9.0-debian-12-r0
```
Comment (alembiewski) [inline licenses.d2iq.yaml:557]:
```suggestion
  - container_image: docker.io/bitnami/postgresql:15.9.0-debian-12-r0
```

**Review by alembiewski — APPROVED** [review]

**Review by alembiewski — CHANGES_REQUESTED** [review]
l

File: services/nkp-insights/1.3.4/defaults/cm.yaml [hunk 3]
```diff
@@ -413,7 +413,7 @@ data:
       image:
         registry: docker.io
         repository: bitnami/postgresql
-        tag: 15.8.0-debian-12-r14
+        tag: 15.9.0-debian-12-r0
```
[the remark this answers is not in the document]
Comment (rajgitnutanix) [reply services/nkp-insights/1.3.4/defaults/cm.yaml]: sure

**Comment by rajgitnutanix** [comment]
> l

Done

**Review by alembiewski — APPROVED** [review]

**Pull request merged**